script/translator.py

```diff
@@ -379,12 +379,68 @@ def translate(self, sl, tl, text, options=None):
         return self._trans
 
 
+class SdcvShell(BasicTranslator):
+    def __init__(self, name="sdcv"):
+        super(SdcvShell, self).__init__(name)
+
+    def get_dictionary(self, sl, tl):
+        """get dictionary of sdcv
+
+        :sl: source_lang
+        :tl: target_lang
+        :returns: dictionary
+
+        """
+        dictionary = ''
+        if (sl == 'en') & (tl == 'zh'):
+            dictionary = '朗道英汉字典5.0'
+        elif (sl == 'zh') & (tl == 'en'):
+            dictionary = '朗道汉英字典5.0'
+        elif (sl == 'en') & (tl == 'ja'):
+            dictionary = 'jmdict-en-ja'
+        elif (sl == 'ja') & (tl == 'en'):
+            dictionary = 'jmdict-ja-en'
+        return dictionary
```

Comment:
i found it exists a problem: if `sl == 'auto'`, it can not match any condition...